src/main/java/CommandManager.java

```diff
@@ -0,0 +1,59 @@
+import java.util.Scanner;
+
+public class CommandManager {
+    private static final Scanner in = new Scanner(System.in);
+    private static final String COMMAND_BYE = "bye";
+    private static final String COMMAND_DONE = "done";
+    private static final String COMMAND_LIST = "list";
+    private static final String COMMAND_TODO = "todo";
+    private static final String COMMAND_DEADLINE = "deadline";
+    private static final String COMMAND_EVENT = "event";
+    private static String userInput;
+
+    private static void setUserInput() {
```

Comment:
```suggestion
    private static void getUserInput() {
```
Perhaps this function can be named getUserInput to better describe the function?